Chart.yaml

```diff
@@ -1,7 +1,7 @@
 # apiVersion v2 is Helm 3
 apiVersion: v2
 name: airflow
-version: 1.13.6
+version: 1.14.0
```

Comment:
Why is this bump needed?

Reply:
0.36 and 0.37 will have different changes right or we are going put same features as before

Reply:
I see, so 1.14 is for 0.37. That makes sense.